Examples/FluxExample.cs

```diff
@@ -6,9 +6,9 @@ namespace Examples
 {
     public static class FluxExample
     {
-        public static async Task Main(string[] args)
+        public static async Task Main()
         {
-            using var fluxClient = FluxClientFactory.Create("http://localhost:8086/");
+            using var fluxClient = new FluxClient("http://localhost:8086/");
```

Comment:
Please rename var to `client`:
```suggestion
            using var client = new FluxClient("http://localhost:8086/");
```